uint32 seq_no
uint32 ack_no
uint16 cmd_code
uint16 param_id
float64 param_val
int16 result_code
string result_msg
